invitations/app_settings.py

```diff
@@ -23,6 +25,14 @@ def CONFIRM_INVITE_ON_GET(self):
         return self._setting('CONFIRM_INVITE_ON_GET', True)
 
     @property
+    def GONE_ON_ACCEPT_ERROR(self):
```

Comment:
I don't love this name, but couldn't come up with something better. Please let me know if you have a better idea!